File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.c
bool findDigit(int n){
    return n==0?false:(int)(log10(abs(n))+1)%2==0;
}

int findNumbers(int* nums, int numsSize) {
    int count = 0;
    for(int i=0;i<numsSize;i++){
        if(findDigit(nums[i])){
            count++;
        }
    }
    return count;
}

File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int i :nums){
            if(findDigit(i)){
                count++;
            }
        }
        return count;
    }

    private static boolean findDigit(int n){
        return n==0?false: ((int) Math.log10(Math.abs(n)) + 1)%2==0;
    }
}